src/form/DecompositionTable/__stubs__/documents.ts

```diff
@@ -0,0 +1,143 @@
+import { ComponentProps } from 'react';
+import DecompositionTable from '../DecompositionTable';
+
+const objectId = '9e267f06-6072-4db9-ba0e-e4adccafa0a9';
+
+export const documents: ComponentProps<typeof DecompositionTable>[ 'rows' ] = [
+	{
+		id: 1,
+		name: 'Hoofddraagconstructie',
+		code: '147',
+		location: 'Amsterdam Centrum',
+		isStructural: false,
+		isElectrical: false,
+		constructionYear: 1979,
+		isArchived: false,
+		isRelevant: true,
+		objectId,
+		actionItems: {
```

Comment:
Should these be statically defined as props? Or would it be better to work with callbacks that at runtime determine what a user is allowed to do?